items/items-perm.md

```diff
@@ -87,21 +89,40 @@ Whether (true/false) the group is the owner of this item. Implies the maximum le
 
-We call *aggregation* the merge of multiple granted permissions to one permission tuple `(group, item)` representing the permissions one group has on an item.
-Those granted permissions may have initially been given to different groups (the group itself or any of its ancestors).
-
-In addition, those granted permissions may have been given from multiple `source groups`,
+We call **aggregation** the merge of all permissions a *group* has on an *item*, granted either:
+- Directly to the group
+- To an ancestor of the group
+
+This notion is generalized to a *user*, with permissions granted either:
+- Directly to the user
+- To any group he is a member of
+- To any ancestor of a group he is a member of
+
+From those granted permissions, here's how the permissions are **aggregated**:
+- `can_view`: we take the maximum among all values
+- `can_grant_view`: we take the maximum among all values
+- `can_watch`: we take the maximum among all values
+- `can_edit`: we take the maximum among all values
+- `is_owner`: if *true* for any granted permission, then *true*. Note that when *true* it makes all above permissions get its maximum possible value.
+- `can_enter_from`:
+  * `NOW()` if it is currently open for any granted permission (if current time is between `can_enter_from` and `can_enter_until`)
+  * otherwise, the sooner date `can_enter_from` of any granted permission located in the future
+  * otherwise, `infinite / never` (year 9999)
+- `can_make_session_official`: if *true* for any granted permission, then *true*
+- `can_request_help_to`: is a *group id* and is not aggregated
+
+Note: those granted permissions that are aggregated may have been given from multiple `source groups`,
 and come from different `origin`.
 See details in the [section on database table permissions_granted](#granted-permissions-table-permissions_granted).
 
-For each permission attribute but *can_make_session_official* and *can_enter_\** (so *can_view*, *can_grant_view*, *can_watch*, *can_edit*, and *is_owner*), we take its **maximum level** among all values. In addition, "is_owner=true" makes every other attribute get its maximum possible value.
-
 ### UI for granting permissions
 
-This image shows the result of **Aggregation** for the permission `can_grant_view` given to `Group` on an `Item`:
-- **Nothing** is the permission granted directly to `Group`
-- **Solution** is the aggregated permission `Group` has, coming from the permissions given to any of its ancestors
-- Note that starting at the third level of permission, **Content**, the text is gray: it means the current-user doesn't have the right to give permissions from this point
+This image shows the result of **Aggregation** for the permission `can_grant_view` given to a `Group` on an `Item`.
+It is viewed from the point of view of a specific `source_group` (the group from which the permission is granted),
+and for a specific `origin` (what process granted the permission).
```

Comment:
Not really. For "group_membership" origin, that's the only permissions that a user (manager) can change

Reply:
fixed